examples/client_timeout_test.c

```diff
@@ -0,0 +1,438 @@
+#include <stdlib.h>
```

Comment:
I have removed this test client for now. I plan on adding support for the timeout call to `client.c` in the future.